src/rpc/server.ts

```diff
@@ -153,20 +190,19 @@ export class Server {
    * Allows you to directly inspect the current state of a contract. This is a
    * backup way to access your contract data which may not be available via
    * events or {@link Server.simulateTransaction}.
+   * Warning: If the data entry in question is a 'temporary' entry, it's
+   *    entirely possible that it has expired out of existence.
    *
    * @param {string|Address|Contract} contract    the contract ID containing the
    *    data to load as a strkey (`C...` form), a {@link Contract}, or an
    *    {@link Address} instance
    * @param {xdr.ScVal} key   the key of the contract data to load
-   * @param {Durability} [durability=Durability.Persistent]   the "durability
+   * @param {Durability} [durability]   the "durability
```

Comment:
Shouldn't we keep the default?

Reply:
Seems it was auto removing the default due to jsdoc/no-defaults. I turned that off and restored the defaults in the docs.